tests/lib/collectors/aws_logging.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests the AWS logging collector."""
+
+import unittest
+
+from dftimewolf.lib import state
+from dftimewolf.lib.collectors import aws_logging
+
+from dftimewolf import config
+
+class AWSLoggingTest(unittest.TestCase):
```

Comment:
_Ideally_, we'd also test if `log_client.LookupEvents` is called as expected (e.g. with the properly formatted times) and that the containers are properly added.